ha/setup/ha_setup.py

```diff
@@ -322,6 +322,11 @@ def process(self):
         self._fetch_fids()
         self._update_cluster_manager_config()
 
+        # Push node name mapping to store
+        if not self._confstore.key_exists(f"{const.PVTFQDN_TO_NODEID_KEY}/{node_name}"):
+            node_id = Conf.get(self._index, f"server_node.{machine_id}.node_id")
+            self._confstore.set(f"{const.PVTFQDN_TO_NODEID_KEY}/{node_name}", node_id)
```

Comment:
const.PVTFQDN_TO_NODEID_KEY}/{node_name}", node_id

I think we need more node map

const.PVTFQDN_TO_NODEID_KEY}/{node_name}" {"hostname": hostname, "node_id":node_id}

can we use json value instead of string

Reply:
@ajay-paratmandali
@ajaysrivas and I had a discussion about this. We do not need to add hostname, instead we will send private_fqdn only in the IEM description as we receive the same from PCS. Hence we need just the private_fqdn to node_id map.